File: Javascript/Programmers_level/level1/없는 숫자 더하기.js
function solution(numbers) {
    let arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
    let answer = 0;
    
    numbers.forEach((item) => {
        const index = arr.indexOf(item)
        arr[index] = 0
    })
             
    arr.forEach((a) => {
        answer += a
    })
    
    return answer;
}

File: python/Programmers_level/level1/없는 숫자 더하기.py
def solution(numbers):
    arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
    stand = set(arr)
    stand2 = set(numbers)
    answer = sum(list(stand - stand2)) # 차집합
    return answer